src/animl/plot_boxes.py

```diff
@@ -0,0 +1,61 @@
+"""
+Module: animl.plot_boxes
+Functionality to draw bounding boxes and labels provided image DataFrame.
+"""
+# pylint: disable=R0914
+# pylint: disable=no-member
```

Comment:
I think we should be careful about disabling pylint bits for the whole module.  Better to do in-line disabling on particular functions or classes etc.  Otherwise we might accidentally mask issues we care about on functions/classes that get added to the module later.

Also: why did you disable no-member? it didn't look to me like pylint was throwing a no-member

Reply:
In-line would only focus for the specific line/function right? I think as long as the module is compliant it wouldn't raise the R0914. But if there are any rather than only the function/class, it might re throw. I put it on the top to catch  to scan through the overall script because there might be cases where it can repeat for multiple functions within the script. 

Also, I disabled the no-member because it did not allow it could not detect the cv2 module I assumed it was throwing an error, can remove it if required.

Reply:
For the cv2 error, I think we should address that in the invocation in pylint rather than hardcoding it into the module.  There are also ways to put in a .pylintrc file so that it targets particularly cv2 so we're not using a shotgun on this.

Yes, the inline disabling would only focus on the particular function. For the R0914, we actually want that to be thrown on additional functions/classes added to this module that exceed the setting and then actively decide ourselves, on a case by case basis, whether we want to disable R0914 on that function/class.  We might be less reluctant to disable if, for instance, a function had 1000 local variables than this case of 17.  In that case, we would be more likely to judge that something is truly wrong in how we have structured our code and that we need to break the function up into sub-functions or something. But we won't even get warned if we disable that at the module level.